script.js

```diff
@@ -1,4 +1,125 @@
+// 2 players: player 1 and player 2
+// 3 game modes:
+// dice roll: is to click submit & show dice rolls.
+// dice order: is to input order of dice & show combined number.
+// show result: is to show which player wins
```

Comment:
Good job being consistent with the pseudo code before writing the code for the game!